crates/utils/re_video/src/demux/mod.rs

```diff
@@ -115,6 +115,27 @@ pub type GopIndex = usize;
 /// Index used for referencing into [`VideoDataDescription::samples`].
 pub type SampleIndex = usize;
 
+/// Distinguishes finate videos from video streams.
+#[derive(Clone)]
+pub enum VideoUpdateType {
```

Comment:
It seems like "delivery method" or "delivery format" is the more common term for this distinction

```suggestion
pub enum VideoDeliveryMethod {
```

Reply:
nice, good suggestion!